### PR TITLE
Add to Lanzano et al. 2019 the adjustment proposed by Lanzano et al. (2022)

### PR DESCRIPTION
**Pull request by mmpagani**

### CODE REVIEW

File: openquake/hazardlib/tests/gsim/lanzano_2019_mod2022_test.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    # Tables provided by original authors
+    MEAN_FILE = "LAN2019/ITA18_MOD22_RJB_MEAN.csv"
+    STD_FILE = "LAN2019/ITA18_MOD22_RJB_STD.csv"
+
+    def test_all(self):
+        self.check(self.MEAN_FILE, self.STD_FILE,
+                   max_discrep_percentage=0.1)
+
+
```
Comment (micheles) [inline openquake/hazardlib/tests/gsim/lanzano_2019_mod2022_test.py:36]: Why are the tests below commented out?

Comment (mmpagani) [reply openquake/hazardlib/tests/gsim/lanzano_2019_mod2022_test.py:36]: I just started the tests using the ones available for the original 2019 version. For the change added this single test would be probably enough. I am waiting for feedback from the authors.

**Comment by mmpagani** [comment]
I contacted the authors and got confirmation that the correction coefficients for PGA can be the ones for SA(0.01) and that the standard deviation can be the one of the original model.

**Review by micheles — APPROVED** [review]

**Comment by micheles** [comment]
Please update the changelog

**Pull request merged**